python/tvm/meta_schedule/relay_integration.py

```diff
@@ -93,8 +93,15 @@ def _normalize_params(
         if isinstance(param, np.ndarray):
             param = nd.array(param)
         relay_params[name] = param
-    if executor is not None:
+
+    if executor is None:
+        executor = relay.backend.Executor("graph")
+
+    if mod.get_attr("executor") is None:
         mod = mod.with_attr("executor", executor)
+    else:
+        executor = mod.get_attr("executor")
```

Comment:
quick question: why do we need the logic on line 97-98? i.e. is non-nullable `executor` useful in subsequent processing?

Reply:
When None `executor` is passed to `relay.build(..., executor=executor)`, we get segfault. This is certainly due to some sloppy coding somewhere, but I haven't looked into it. 

Perhaps the default value of `executor` should be `Executor("graph")` rather than None, as in `relay.build` https://github.com/apache/tvm/blob/main/python/tvm/relay/build_module.py#L281

Reply:
I see! This is certainly annoying that it’s engineered this way…Thanks for your explanation!

We probably don’t need to over-engineer it for now. Let’s keep it as it is given vm is not supported.